ERCS/erc-7888.md

```diff
@@ -2,12 +2,13 @@
 eip: 7888
 title: Crosschain Broadcaster
 description: Trustless crosschain storage-based message broadcasting for rollups.
-author: Henry Arneson (@godzillaba), Chris Buckland (@yahgwai)
+author: Henry Arneson (@godzillaba), Chris Buckland (@yahgwai), Ernesto García (@ernestognw)
 discussions-to: https://ethereum-magicians.org/t/new-erc-cross-chain-broadcaster/22927
 status: Draft
 type: Standards Track
 category: ERC
 created: 2025-02-18
+requires: 7786
```

Comment:
It should be https://github.com/ethereum/ERCs/pull/1072 instead once it has a number, will update then